### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by guyfe**

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ Our hardware specification recommendations based on code size:
 - The [flows-cli](./flows-cli.sh) script.
 - `realpath` (some older Mac distributions do not contain it - use [homebrew coreutils](https://formulae.brew.sh/formula/coreutils))
 
+#### Docker Desktop configuration
+See screenshot below:
+-  Use Virtualization framework should be truned on
```
Comment (nerya10) [inline cli/README.md:28]: Use Virtualization framework - should be truned on

Comment (david-piiano) [inline cli/README.md:28]:
```suggestion
-  `Use Virtualization framework` setting should be enabled
```

File: cli/README.md [hunk 2]
```diff
@@ -23,6 +23,15 @@ Our hardware specification recommendations based on code size:
...
+-  File sharing option - select VirtioFS
```
Comment (david-piiano) [inline cli/README.md:29]:
```suggestion
-  `File sharing implementation` - select `VirtioFS` 
```

File: cli/README.md [hunk 3]
```diff
@@ -23,6 +23,15 @@ Our hardware specification recommendations based on code size:
 - The [flows-cli](./flows-cli.sh) script.
 - `realpath` (some older Mac distributions do not contain it - use [homebrew coreutils](https://formulae.brew.sh/formula/coreutils))
 
+#### Docker Desktop configuration
+See screenshot below:
+-  Use Virtualization framework should be truned on
+-  File sharing option - select VirtioFS 
+-  Use Rosetta for x86/amd64 emulation on Apple Silicon - should be turned off
```
Comment (david-piiano) [inline cli/README.md:30]:
```suggestion
-  Users of Apple Silicon - disable the usage of Rosetta for x86/amd64 emulation 
```

File: cli/README.md [hunk 4]
```diff
@@ -23,6 +23,15 @@ Our hardware specification recommendations based on code size:
 - The [flows-cli](./flows-cli.sh) script.
 - `realpath` (some older Mac distributions do not contain it - use [homebrew coreutils](https://formulae.brew.sh/formula/coreutils))
 
+#### Docker Desktop configuration
+See screenshot below:
+-  `Use Virtualization framework` setting should be enabled
+-  `File sharing implementation` - select `VirtioFS` 
+-  Users of Apple Silicon - disable the usage of Rosetta for x86/amd64 emulation
```
Comment (david-piiano) [inline cli/README.md:30]:
```suggestion
-  For the Apple Silicon users - uncheck  `Use of Rosetta for x86/amd64 emulation on Apple Silicon`
```

**Review by david-piiano — APPROVED** [review]

**Pull request merged**